plugin.rb

```diff
@@ -84,5 +85,37 @@
         end
       end
     end
+
+    add_automation_scriptable("send_chat_integration_message") do
+      field :provider,
+            component: :choices,
+            extra: {
+              content:
+                DiscourseChatIntegration::Provider.enabled_provider_names.map do |provider|
+                  { id: provider, name: "chat_integration.provider.#{provider}.title" }
+                end,
+            },
+            required: true
+      field :channel_name, component: :text, required: true
+
+      version 1
+
+      triggerables %i[topic_tags_changed]
+
+      script do |context, fields, automation|
+        provider = fields.dig("provider", "value")
+        channel_name = fields.dig("channel_name", "value")
+
+        post = DiscourseChatIntegration::ChatIntegrationReferencePost.new(context)
```

Comment:
Im not sure we should pass the context directly here, maybe be more specific on what you pass to `DiscourseChatIntegration::ChatIntegrationReferencePost.new`, the API of automation might send different things in the future